srcpkgs/ripright/template

```diff
@@ -0,0 +1,14 @@
+# Template file for 'ripright'
+pkgname=ripright
+version=0.11
+revision=1
+build_style=gnu-configure
+hostmakedepends="pkg-config"
+makedepends="libflac-devel libmagick6-devel libcurl-devel libdiscid-devel libcdparanoia-devel"
+depends="libflac libmagick6 libcurl libdiscid libcdparanoia"
```

Comment:
these libraries are automatically found via shlibs, remove

Reply:
good to know, thanks! Must have missed that part in the manual